sso-plugin/optimize-sso-keycloak/elasticsearch/Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM docker.elastic.co/elasticsearch/elasticsearch-oss:6.0.0
+ENV ES_JAVA_OPTS="-Xms512m -Xmx512m -Djava.io.tmpdir=/usr/share/elasticsearch/tmp"
+ADD elasticsearch.yml /usr/share/elasticsearch/config/
+USER root
+RUN mkdir /usr/share/elasticsearch/tmp
+RUN chown elasticsearch:elasticsearch /usr/share/elasticsearch/tmp
+RUN mkdir /usr/share/elasticsearch/config/scripts
```

Comment:
this dir is not used? did you try adding the license with a custom script? ^^

Reply:
removed that.